tests/check/signed_video_helpers.c

```diff
@@ -20,15 +20,26 @@
  */
 #include "signed_video_helpers.h"
 
+#include <assert.h>  // assert
 #include <check.h>
-#include <stdlib.h>  // calloc, size_t
+#include <stdlib.h>  // size_t
 
 #include "lib/src/includes/signed_video_common.h"
 #include "lib/src/includes/signed_video_openssl.h"
 #include "lib/src/includes/signed_video_sign.h"
 #include "lib/src/signed_video_h26x_internal.h"  // signed_video_set_recurrence_interval()
 #include "lib/src/signed_video_internal.h"  // _signed_video_t
 
+#define RSA_PRIVATE_KEY_ALLOC_BYTES 2000
+#define ECDSA_PRIVATE_KEY_ALLOC_BYTES 1000
+
+char global_private_key_rsa[RSA_PRIVATE_KEY_ALLOC_BYTES];
```

Comment:
One potential naming improvement.
`global_private_key_rsa` -> `private_key_rsa`
and same for ecdsa.

No need for `global` I think.